packages/commonjs/src/base-cjs-module-system.ts

```diff
@@ -25,7 +25,7 @@ export interface IBaseModuleSystemOptions {
    *
    * @returns resolved path, or `undefined` if cannot resolve.
    */
-  resolveFrom(contextPath: string, request: string, requestOrigin?: string): string | undefined;
+  resolveFrom(contextPath: string, request: string, requestOrigin?: string): string | false | undefined;
```

Comment:
explain in jsdocs what is the difference between false and undefined

Reply:
will do